cmd/vclusterctl/cmd/completion.go

```diff
@@ -0,0 +1,92 @@
+package cmd
+
+import (
+	"errors"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+const compDesc = `
+#######################################################
+################### vcluster complete #################
```

Comment:
Please rename to vcluster completion as the sub command is called completion as well